### PR TITLE
update readme.

### PR DESCRIPTION
**Pull request by DanielBarros1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# project-doggo
\ No newline at end of file
+# project-doggo
+
+#amaze
```
Comment (barakyoresh) [inline README.md]: missing whitespace.

**Comment by barakyoresh** [comment]
Round 1.

File: README.md (hunk 1 above)
Comment (DanielBarros1) [inline README.md]: Fixes.

**Comment by DanielBarros1** [comment]
Round 1 complete.

**Comment by barakyoresh** [comment]
squash.

**Comment by DanielBarros1** [comment]
Squashed.

**Comment by barakyoresh** [comment]
CC @eliranswisa Enjoy new feature!

**Pull request merged**